poly-synth/src/poly-synth.js

```diff
@@ -103,15 +115,50 @@ class PolySynth {
     let reverbInputNode = new GainNode(this.context_);
     let reverbOutputNode = this.createReverbGraph_(reverbInputNode);
 
-    // The client is responsible for connecting |this.output| to a destination.
-    this.output = new GainNode(this.context_);
-
-    this.voiceOutput_.connect(bitcrusherInputNode);
+    this.voiceOutput_.connect(noisegateInputNode);
+    noisegateOutputNode.connect(bitcrusherInputNode);
     bitcrusherOutputNode.connect(delayInputNode);
     delayOutputNode.connect(reverbInputNode);
     reverbOutputNode.connect(this.output);
   }
 
+  createNoisegateSideChainGraph_(inputNode) {
```

Comment:
I really think there should be two inputs, one for the signal and one for the side chain control, but too late for that now, I think.